### ISSUE TITLE
ROKS calculation of open-singlet molecule

### ISSUE THREAD

**Issue by proexcuse** [pyscf/pyscf#931]
Hi,

I'm interested in a open-singlet molecule (two open orbitals with singlet total spin. )
Is it possible that roks calculation converges to open singlet?

Using get_occ method in pyscf/scf/rohf.py, it seems the results always converge to closed singlet. But is there any way for ROKS calculation of open-singlet?

**Comment by proexcuse** [comment]
Let me share an example of an open-singlet molecule. 
https://pubs.acs.org/doi/10.1021/ja030060f

**Comment by susilehtola** [comment]
> I'm interested in a open-singlet molecule (two open orbitals with singlet total spin. )
> Is it possible that roks calculation converges to open singlet?

If ROKS means same orbitals for different spin, having an open shell singlet is not possible by definition.

**Comment by proexcuse** [comment]
You are right. The electronic structure I mentioned leads to spin contamination. 

Thanks.
